### PR TITLE
feat(base.py.Fitness): check whether len(values) == len(weights)

### PR DESCRIPTION
**Pull request by yxliang01**
This check can make many careless errors visible.

### CODE REVIEW

**Comment by yxliang01** [comment]
@fmder It has been 5 months already. Are you interested in reviewing this?

File: deap/base.py [hunk 1]
```diff
@@ -185,6 +185,7 @@ def getValues(self):
         return tuple(map(truediv, self.wvalues, self.weights))
 
     def setValues(self, values):
+        assert len(values) == len(self.weights)
```
Comment (fmder) [inline deap/base.py]:
```suggestion
        assert len(values) == len(self.weights), "Assigned values have not the same length than fitness weights"
```

**Comment by fmder** [comment]
LOL, you are right! 5 months for a one line commit! Let

**Comment by yxliang01** [comment]
@fmder Don

**Comment by yxliang01** [comment]
@fmder  Looks like you are the only active maintainer with write permission. Maybe can directly merge? :joy:

**Comment by cyasing** [comment]
> This check can make many careless errors visible.

Could you explain what kind of errors does this catch? I am trying fitnessMulti type for 2 objectives (weights=(-1.0, 1.0)), and 5 inputs to be optimized.

**Pull request merged**